usage/rest-api/src/main/java/brooklyn/rest/api/ApplicationApi.java

```diff
@@ -76,6 +76,12 @@ public JsonNode fetch(
             value = "Fetch list of applications, as ApplicationSummary objects",
             responseClass = "brooklyn.rest.domain.ApplicationSummary"
     )
+    public List<ApplicationSummary> list(
+            @ApiParam(value = "Regular expression to filter by", required = false)
+            @DefaultValue(".*")
+            @QueryParam("typeRegex") String typeRegex);
+
+    /** As {@link #list(String)}, filtering for <code>.*</code>. */
     public List<ApplicationSummary> list();
```

Comment:
I guess you are keeping the method for backwards compatibility with rest-client? But in this case it should have it's own set of annotations, so that client calls know what to do.